ledger/ledger-api-integration-tests/BUILD.bazel

```
@@ -44,6 +44,7 @@ dependencies = [
     "//ledger-api/rs-grpc-akka",
     "//ledger-api/rs-grpc-bridge",
     "//ledger-api/testing-utils",
+    "//ledger/ledger-api-auth:ledger-api-auth",
```

```suggestion
    "//ledger/ledger-api-auth",
```
Not sure of why we don't do this also in the rest of the file, feel free to put it there as well. :slightly_smiling_face:

---

This would be a perfect task for our `./fmt.sh` script